cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -42,7 +45,22 @@ export KUBEVIRTCI_GOCLI_CONTAINER=quay.io/kubevirtci/gocli:latest
 
   # Run conformance test only at CI and if the provider has them activated
   conformance_config=$DIR/${provision_dir}/conformance.json
+
   if [ "${CI}" == "true" -a -f $conformance_config ]; then
+    sonobouy_plugins=""
+
+    if [ "$RUN_KUBEVIRT_CONFORMANCE" == "true" ]; then
+      LATEST=$(curl -L "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/latest")
+      ${ksh} apply -f "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/kubevirt-operator.yaml"
+      ${ksh} apply -f "https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/kubevirt-cr.yaml"
+
+      ${ksh} wait -n kubevirt kv kubevirt --for condition=Available --timeout 15m
+
+      export SONOBUOY_EXTRA_ARGS="--plugin https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/conformance.yaml"
+      hack/conformance.sh $conformance_config
```

Comment:
Would it makes ense to only add `--plugin https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${LATEST}/conformance.yaml` to `SONOBUOY_EXTRA_ARGS` and basically running below then:

```
--plugin systemd-logs --plugin e2e --plugin https://<url>/conformance.yaml
```

Reply:
Let's ensure that we run sonobuoy only once, since we also check the results only once. Otherwise you don't benefit from the early exit which you want after the kubevirt tests. The rest looks very good.

Reply:
My intention was run kubevirt conformance on a different "sonobuoy session"
where kubevirt conformance runs first (currently it takes less time).
so a job could fail faster in case of an error and enable us to control it with `RUN_KUBEVIRT_CONFORMANCE`

Reply:
I am not sure I fully understand this. Let me try to summarize like this what I would expect:
 * It is fine if you want/can somehow influence with env vars what to run and to fail fast, just ensure that
 * by default all three plugins are run in CI
 * that a failing kubevirt conformance run really leads to exiting the script early with a non-nil exit code (I am not out of the box sure if sonobuoy returns with a non-0 return code and that we exit the script). To really fail fast, like you suggest.

If that is the behaviour which we get, all is good from my perspective.

Reply:
It should cover what you expect: 

> * It is fine if you want/can somehow influence with env vars what to run and to fail fast, just ensure that

Control running kubevirt-conformance with `RUN_KUBEVIRT_CONFORMANCE` if its set to false,
only the standard test will run ( e2e and systemd-logs)

> * by default all three plugins are run in CI (if previous plugins don't fail, I don't mind early exit)

This PR configures all tree to run as we set `RUN_KUBEVIRT_CONFORMANCE` to be true by default on the last commit.

> * that a failing kubevirt conformance run really leads to exiting the script early with a non-nil exit code (I am not out of the box sure if sonobuoy returns with a non-0 return code and that we exit the script). To really fail fast, like you suggest.

As we mentioned on other thread the caller script, check-cluster-up.sh, will runs conformance.sh twice:
first for kubevirt-conformance tests and the second for the standard tests (e2e, systemd-logs).

On `conformance.sh `script we use trap on EXIT and calling to `teardown`.
`teardown` check the tests status with `sonobouy status`, 
in case there were failing tests  `conformance.sh ` exit with error code 1.

The error will propagate to `check-cluster-up.sh` script, which runs with `-eo pipefail` flags,
propagating the erorr forwarder.

If we will find that running it this way is not best suite for us
we can always run all the tests in one shot (by setting all plugin to the same sonobuoy run)